napari/utils/status_messages.py

```diff
@@ -60,9 +60,11 @@ def generate_layer_status(name, position, value):
     msg : string
         String containing a message that can be used as a status update.
     """
-    full_coord = map(str, np.round(position).astype(int))
-
-    msg = f"{name} [{' '.join(full_coord)}]"
+    if position is not None:
+        full_coord = map(str, np.round(position).astype(int))
+        msg = f"{name} [{' '.join(full_coord)}]"
+    else:
+        msg = f"{name}"
```

Comment:
we may want to update the var name and docstring here as it no longer just takes the layer name